Automattic/woocommerce-payments
# fix: payment method icon alt text

Fixes #

#### Changes proposed in this Pull Request









While working on https://github.com/Automattic/woocommerce-payments/issues/9625 , I noticed that the `alt` text on the payment method icon displayed at checkout contains some undesired markup.

Before:
<img width="2106" alt="Screenshot 2024-10-30 at 4 53 18 PM" src="https://github.com/user-attachments/assets/15ddeed3-ea78-4168-81fe-0a076771805c">

After:
<img width="1291" alt="Screenshot 2024-10-30 at 4 52 19 PM" src="https://github.com/user-attachments/assets/8e568cb3-ac70-4257-a5a6-76e4058c7a35">


#### Testing instructions





- As a customer, add some products to the cart
- Navigate to the checkout
- Inspect the `card` payment method's icon
- Its `alt` text should no longer contain markup, it should instead be `Credit card / debit card payment method logo`
  - I added the " payment method logo" text to the end of the `alt` tag, because I think it's more descriptive 🤷 
- The same text should be present on the `alt` attribute for the other WooPayments payment method icons at checkout

-------------------


**Post merge**
